scripts/prep_hofx3d_yaml.sh

```diff
@@ -30,12 +30,16 @@ done
 shift $((OPTIND -1))
 
 cd ${RUNDIR}
+echo "RUNDIR=" $RUNDIR
 
 ## Set date for hofx
 sed -i "s/WINDOW_BEGIN/${window_begin}/g" ${yamlfile}
 sed -i "s/WINDOW_LENGTH/${window_length}/g" ${yamlfile}
 sed -i "s/BKG_DATE/${bkg_date}/g" ${yamlfile}
 sed -i "s/MEMBER_NO/${mbr}/g" ${yamlfile}
+sed -i "s/ocn.pert.ens..nc/MOM.res.nc/g" ${yamlfile}
+sed -i "s/cic.pert.ens..nc/cice_bkg.nc/g" ${yamlfile}
```

Comment:
why do you need perturbations? Are you looking at an ensemble of hofx?

Reply:
The perturbations are part of the ensemble runs from @flampouris.
I did not modify it and also. the script needs a 3rd argument "mbr" for ensembles.

@flampouris need your suggestion on this.

Reply:
This copying is happening because the letkf uses the Hofx job. Keep it in, and i will modify it at the next PR.